x/auth/types/txbuilder.go

```
@@ -74,6 +76,71 @@ func NewTxBuilderFromCLI(input io.Reader) TxBuilder {
 	return txbldr
 }
 
+// NewTxBuilderFromCLI returns a new initialized TxBuilder with parameters extracted from
+// FlagSet (It should deprecate NewTxBuilderFromCLI).
+func NewTxBuilderFromFlagSet(input io.Reader, fs *pflag.FlagSet, keyringPath string) (TxBuilder, error) {
+	backend, err := fs.GetString(flags.FlagKeyringBackend)
```

I've been ignoring any errors returned here. It's OK to do IMHO.